setup.py

```diff
@@ -31,39 +19,38 @@
 
 # TODO: find a way to keep this in sync with requirements.txt
 requires = [
-    'Twisted==16.0.0',
-    'Yapsy==1.11.223',
-    'appdirs==1.4.0',
-    'argparse==1.2.1',
-    'colorama==0.3.7',
-    'dnspython==1.12.0',
-    'ecdsa==0.13',
-    'envparse==0.2.0',
-    'gmpy==1.17',
-    'jsonrpc==1.2',
-    'jsonrpclib==0.1.7',
-    'jsonschema==2.5.1',
-    'lbryum>=2.7.4',
-    'loggly-python-handler==1.0.0',
-    'miniupnpc==1.9',
-    'pbkdf2==1.3',
-    'protobuf==3.0.0',
-    'pycrypto==2.6.1',
-    'qrcode==5.2.2',
-    'requests==2.9.1',
-    'requests_futures==0.9.7',
-    'seccure==0.3.1.3',
-    'simplejson==3.8.2',
-    'six>=1.9.0',
-    'slowaes==0.1a1',
-    'txJSON-RPC==0.5',
-    'wsgiref==0.1.2',
-    'zope.interface==4.1.3',
-    'base58==0.2.2',
-    'googlefinance==0.7',
-    'pyyaml==3.12',
-    'service_identity==16.0.0',
-    'ndg-httpsclient==0.4.2',
+    'Twisted',
+    'appdirs',
+    'argparse',
+    'colorama',
+    'dnspython',
+    'ecdsa',
+    'envparse',
+    'gmpy',
+    'jsonrpc',
+    'jsonrpclib',
+    'jsonschema',
+    'lbryum',
+    'loggly-python-handler',
+    'miniupnpc',
+    'pbkdf2',
+    'protobuf',
+    'pycrypto',
+    'qrcode',
+    'requests',
+    'requests_futures',
+    'seccure',
+    'simplejson',
+    'six',
+    'slowaes',
+    'txJSON-RPC',
+    'wsgiref',
+    'zope.interface',
+    'base58',
+    'googlefinance',
+    'pyyaml',
+    'service_identity',
+    'ndg-httpsclient',
```

Comment:
Even this is is not considered best practice.  setup.py should only list the dependencies that lbrynet needs (not a list of dependencies of dependencies) along with any known version restrictions.  Its generally a bad idea to pin versions in setup.py

On the other hand - requirements.txt is intended to be an explicit set of libraries for deployment.

https://caremad.io/posts/2013/07/setup-vs-requirement/
https://www.reddit.com/r/Python/comments/3uzl2a/setuppy_requirementstxt_or_a_combination/cxjs7n4/?utm_content=permalink&utm_medium=front&utm_source=reddit&utm_name=Python